src/main/java/org/isf/lab/rest/LaboratoryController.java

```diff
@@ -404,7 +404,7 @@ public ResponseEntity<List<LabWithRowsDTO>> getLaboratory(@PathVariable Integer
 		}
 
 		List<Laboratory> labList = laboratoryManager.getLaboratory(patient).stream()
-				.filter(e -> !e.getStatus().equals(LaboratoryStatus.DRAFT.toString())).collect(Collectors.toList());
+				.filter(e -> !e.getStatus().equals(LaboratoryStatus.DRAFT.toString()) && !e.getStatus().equals(LaboratoryStatus.OPEN.toString())).collect(Collectors.toList());
```

Comment:
LaboratoryStatus.OPEN, DRAFT, etc are all enums.   Why not define class level statics here for the `toString()` values instead regenerating them each time they are need?